src/res/jamulus.desktop

```diff
@@ -2,8 +2,8 @@
 Name=Jamulus
 Comment=Jam Session
 GenericName=Internet Jam Session Software
-Exec=jamulus
-Icon=jamulus
+Exec=Jamulus
+Icon=Jamulus
```

Comment:
Jamulus can be configured to use a lower case binary. This can be done via qmake command line argument. Since usually Linux tools are with lower letter, maybe for the Debian package you should also make use of that option.
Another thing: What about moving this file in the new distributions file? Or do you think the current location is ok?

Reply:
this is no problem. Shall I or have you already changed that?

Reply:
Do you mean the lower case binary of the movement of the file? You can find the change for the lower case name here: "https://github.com/corrados/jamulus/commit/127e9ac7866d5b750a3b17568d24bc81aa2f192f#diff-ff31b6d635c626acb697d69ef0e864bd".
The file location I did not change. Feel free to move it if you like.

Reply:
It seems that this does not have an effect on the filename. As far as I know qmake takes the name for the final binary from the Jamulus.pro file name, so you (or I) would change it to lower case then the binary would also be lower case. At least on debian and ubuntu it worked that way. So if you change that, the destination file will be lower case. Two things why I think it would be better if you'd do that: i can not say if it has any impact on the windows build. i can not say if it has any impact on the OSX build. So I could get a appropriate build-env on Windows and I don't own a Mac / X-Code

Reply:
i just realized that you probably want to intentionally have uppercase J for Windows / OSX. i did now use the parameter you gave me: https://github.com/braindef/jamulus/blob/master/distributions/debian/rules#L6